subprojects/docs/src/docs/release/notes.md

```diff
@@ -112,6 +112,9 @@ The previously deprecated support for Play Framework 2.2 has been removed.
 - Forbid passing `null` as configuration action to the methods `from` and `to` on `CopySpec`.
 - Removed the property `bootClasspath` from `CompileOptions`.
 - Registering invalid inputs or outputs via the runtime API is now an error.
+- Chaining calls to the methods `file`, `files`, and `dir` on `TaskInputs` is now impossible.
+- Chaining calls to the methods `file`, `files`, and `dir` on `TaskOutputs` is now impossible.
```

Comment:
There's also `dirs()` on `TaskOutputs`.

Reply:
`dirs` was added in `3.3` and was never chainable.

Reply:
👍